examples/02_model_collaborative_filtering/multi_vae_deep_dive.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>This section is taken almost verbatim from the paper. Please summarize in your own words and link to the relevant section in the paper for further reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>